src/api.js

```diff
@@ -14,6 +14,12 @@ export const getArticleById = (id) => {
     })
 }
 
+export const getAllCommentFromId = (id) => {
+    return axios.get(baseUrl+'articles/'+id+'/comments').then(({data}) => {
```

Comment:
You should look at axios.create that could really simplify your handling of the base url